src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  background-color: lightgrey;
+}
+
+.logo {
+  line-height: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 40px 0;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.navbar__item {
+  margin-right: 20px;
+  position: relative;
+}
+
+.navbar__item:last-child {
+  margin-right: 0;
+}
+
+.navbar__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding-inline-start: 0;
+}
+
+.navbar__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
```

Comment:
move this rule to html tag

Reply:
> move this rule to html tag
what exactly rule must i move to html tag?